### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by fflorent**
## Problem

- Manifest v2

## Solution

- *And how do you fix that problem*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by fflorent** [comment]
@Salamandar I made some changes. Tell me if you want to review it.

**Comment by ericgaspar** [comment]
!testme

File: doc/POST_INSTALL.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Congratulation, Open Search is installed!
+
+You may try its API right now. Try this command:
+curl http://localhost:**PORT**
```
Comment (Salamandar) [inline doc/POST_INSTALL.md:4]:
```suggestion
`curl http://localhost:__PORT__`
```

File: scripts/install [hunk 2]
```diff
@@ -47,11 +47,11 @@ chown "$app:$app" "$install_dir/config/opensearch.yml" "$install_dir/config/jvm.
 #=================================================
 ynh_script_progression --message="Increasing maximum map count (sysctl)..."
 
-# Increase the maximum number of files inotify can monitor.
-ynh_add_config --template="90-max_map_count-opensearch.conf" --destination="/etc/sysctl.d/"
+if [ "${container:-}" != "lxc" ]; then # lxc doesn't allow sysctl to play with kernel options.
```
Comment (Salamandar) [inline scripts/install:50]: Ah yes, you’re right here indeed :D

**Review by Salamandar — APPROVED** [review]
Just a small comment on the doc, but I agree with your scripts modifications.

File: doc/POST_INSTALL.md (hunk 1 above)
Comment (fflorent) [reply doc/POST_INSTALL.md:4]: Thanks!

**Pull request merged**